### ISSUE TITLE
Layout inconsistency

### ISSUE THREAD

**Issue by darealshinji** [tedsmith/quickhash#11]
The layout of the tabs is not always consistent.
In "Compare Two Files" and "Compare Directories" the  "Hash Algorithm" group is within another group while on the other tabs it's on its own. The group title font sizes also differ on the first two tabs. And the font size is sometimes too big so part of the shown text is hidden. For example for me it's "Hash Algorit". The positioning of the groups is also a bit inconsistent between the tabs.

**Comment by tedsmith** [comment]
See comments in #12  Thanks again Dareal.

**Comment by tedsmith** [comment]
Made to some minor changes to your changes. The "FileS" tab was reverted back to being called that, as I notice you'd changed it to "Multiple Files". Your change makes sense of course, and indeed it used to be called that a few years ago. But when I begain streamlining the interface, several users made an argument along the lines of "We know each tab does some sort of hashing, so if "File" is for a file, then we don't need to see "Multiple Files" or "Several Files"...just use "FileS". So I did, and the user manual has been re-written for that. It's a too bigger job to revert it all again, so I've reverted the tab back to "FileS".
